### ISSUE TITLE
Conversion Queue status from xml doesn't fail gracefully when Queue status is unavailable.

### ISSUE THREAD

**Issue by troydai** [Azure/azure-storage-python#294]
Occasionally, the Queue service returns unavailable status:
```XML
<?xml version="1.0" encoding="utf-8"?><StorageServiceStats><GeoReplication><Status>unavailable</Status><LastSyncTime></LastSyncTime></GeoReplication></StorageServiceStats>
```
Because of the missing last sync time, deserialization keep throwing `TypeError` which not only not representing the correct service feedback, but also keeps SDK retrying muliple times.
```
https://github.com/Azure/azure-storage-python/blob/c9655db4aac42333f304d60d6d909a9579ed4474/azure/storage/_deserialization.py#L187
```

**Comment by rambo-msft** [comment]
Thanks @troydai, I will address this in our next release.

**Comment by zezha-msft** [comment]
@troydai  This has been fixed in the new release [0.35.0](https://github.com/Azure/azure-storage-python/releases/tag/v0.35.0).
